content/concepts/module-resolution.md

```diff
@@ -24,26 +24,26 @@ The dependency module can be from the application code or a third party library.
 * Absolute paths
 
 ```js
-require("/home/me/file");
-require("C:\\Home\\me\\file");
+import "/home/me/file";
+import "C:\\Users\\me\\file";
 ```
 
 Since we already have the absolute path to the file, no further resolution is required.
 
 * Relative paths
```

Comment:
The formatting here feels weird. These probably should be subheaders (`###` most likely).